client/js/app.js

```diff
@@ -70,6 +71,17 @@ const init = () => {
       cssPrefix: 'numbered-list__'
     });
   }
+
+  if (galleries) {
```

Comment:
As a point here - this check is superfluous - we could just say:
`nodeList` never returns a falsey - but does return an empty array, which means the `forEach` won't run.